libs/libmbim/Makefile

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (C) 2016 Velocloud Inc.
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libmbim
+PKG_VERSION:=1.18.2
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://anongit.freedesktop.org/libmbim/libmbim
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=2028fd2f2effbcf80d0503cc163ac8f6316d8359
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
+
+PKG_MAINTAINER:=Nicholas Smith <nicholas.smith@telcoantennas.com.au>
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
```

Comment:
Please try compiling with PKG_BUILD_PARALLEL:=1. If it works, add it here.

Reply:
That should definitely work